sklearn/metrics/_ranking.py

```diff
@@ -425,23 +425,23 @@ class scores must correspond to the order of ``labels``,
         computation currently is not supported for multiclass.
 
     multi_class : {'raise', 'ovr', 'ovo'}, default='raise'
-        Multiclass only. Determines the type of configuration to use. The
+        This implementation can be used with multiclass classification only. Determines the type of configuration to use. The
         default value raises an error, so either ``'ovr'`` or ``'ovo'`` must be
         passed explicitly.
 
         ``'ovr'``:
-            Computes the AUC of each class against the rest [3]_ [4]_. This
+            Stands for One-vs-rest algorithm. Computes the AUC of each class against the rest [3]_ [4]_. This
```

Comment:
For this one and for ovo, I'm personally not sure we need to explicitly mention the full name. There are many places where we just write 'ovo' and 'ovr', I guess this is common knowledge.

If we want to keep it, I'd still remove "algorithm".